tests/MulticastUDPClient.py

```
@@ -423,16 +423,16 @@
-	sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP)
-	tsts_fxr = MCastClient()
-	print(str((HOST, PORT)))
-	tsts_fxr.say(HOST, PORT, sock, data)
-	tsts_fxr.say(HOST, PORT, sock, str("STOP"))
+	HOST, PORT = "224.0.0.1", 59991  # pragma: no cover
+	data = "TEST This is a test"  # pragma: no cover
+	sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP)  # pragma: no cover
+	tsts_fxr = MCastClient()  # pragma: no cover
+	print(str((HOST, PORT)))  # pragma: no cover
+	tsts_fxr.say(HOST, PORT, sock, data)  # pragma: no cover
+	tsts_fxr.say(HOST, PORT, sock, str("STOP"))  # pragma: no cover
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)